tests/cdi/cdi_test.go

```diff
@@ -0,0 +1,46 @@
+package cdi_test
+
+import (
+	"flag"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Importing and starting a VM using CDI", func() {
+	flag.Parse()
+	ktests.SkipIfNoOc()
+	dstPVCFilePath := "/tmp/test-pvc.json"
+	dstVMFilePath := "/tmp/test-vm.json"
+
+	url, ok := os.LookupEnv("STREAM_IMAGE_URL")
+	if !ok {
+		url = pvcEPHTTPNOAUTHURL
+	}
+
+	Context("PVC with valid image url will succeed", func() {
+		It("should create the PVC successfully", func() {
+			tests.ProcessTemplateWithParameters(rawPVCFilePath, dstPVCFilePath, "PVC_NAME="+pvcName, "EP_URL="+url)
+			tests.CreateResourceWithFilePathTestNamespace("pvc", pvcName, dstPVCFilePath)
+		})
+
+		Specify("the PVC should become bound", func() {
+			tests.WaitUntilResourceReadyByNameTestNamespace("pvc", pvcName, "-o=jsonpath='{.metadata.annotations}'", "pv.kubernetes.io/bind-completed:yes pv.kubernetes.io/bound-by-controller:yes")
+		})
+
+		Specify("the importer-pod should become completed", func() {
+			tests.WaitUntilResourceReadyByLabelTestNamespace("pod", "app=containerized-data-importer", "-o=jsonpath='{.items[0].status.phase}'", "Succeeded")
```

Comment:
how do you know the 1st pod listed (matching the label) is the pod you're interested in? I think this test is too imprecise.

Reply:
because I only created one and I didn't find any other labels to match more imprecisely. do you have better suggestions? if not maybe we should create an issue for CDI repo, to add more precise labels to importer-pod

Reply:
I was thinking that maybe namespace (which is random-ized) would be a good additional filter. But if you have full control of the test env such that you know no other cdi tests will run in parallel with this test then what you have here is fine. My comment is not about the value of the label but that there can potentially be >1 importer pod if more than one cdi test is run at the same time.

Reply:
thank you for the suggestion, I will create an issue for that about the randomized namespace and more than one pod conflicts, will do that in a follow-up pr